pysat/instruments/methods/general.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""Provides generalized routines for integrating instruments into pysat.
```

Comment:
Thanks for starting this. Eventually we may want to move/create some generalized download and list_remote_file methods in here.